examples/using-promise-actions/README.md

```diff
@@ -0,0 +1,13 @@
+# Using Promise Middleware
```

Comment:
the title should be something like "Using Promise Middleware with ..." and the description should briefly explain the nature/purpose of the example.